docs/developer/decision-records/2022-06-16-distributed-authorization/distributed-authorization.puml

```diff
@@ -8,7 +8,9 @@ end box
 box "Participant B" #LightGreen
 Participant "Participant B\nAPI" as B
 end box
-
+box "Authority"
+Participant "Authority\nDID Document" as Auth
+end box
 ParticipantA -> B ++: Request:\n - DID JWS\n - Hub Token
```

Comment:
Identity Hub authorization has been outscoped. I think we should remove the "Hub Token" from this diagrams. @algattik

Reply:
So instead of `Get credentials: -Hub token` we would have only `Get Verifiable Credentials' without the  box _authorize request_ ?

Reply:
Exactly, that is the current state of the design (and valid for both diagrams). Authorization will come at a later point in time, and it is still yet to be defined how it will work exactly.

Reply:
makes sense, I simplified the flow

Reply:
@Izzzu As per this discussion above feels like we want to remove HubToken from it, but HubToken is still mentioned in puml/png , for example in step 1 it sends time-limited bearer token (hub token)